blazor-toc.html

```diff
@@ -1499,9 +1499,18 @@
 					<li>
 						<a href="/blazor/datagrid/how-to/css-isolation-for-grid">CSS Isolation for DataGrid</a>
 					</li>
-					<li>
+                	<li>
 						<a href="/blazor/datagrid/how-to/use-radio-button-instead-of-checkbox">Use radio button instead of checkbox in single selection mode of Grid</a>
 					</li>
+					<li>
+						<a href="/blazor/datagrid/how-to/hide-checkbox-in-column-header">Hide CheckBox in Column Header</a>
+					</li>
```

Comment:
move these topics as suggested in this PR https://github.com/syncfusion-content/blazor-docs/pull/622